services/motion/builtin/builtin.go

```diff
@@ -237,7 +270,118 @@ func (ms *builtIn) MoveOnGlobe(
 	angularVelocity float64,
 	extra map[string]interface{},
 ) (bool, error) {
-	return false, ErrNotImplemented
+	operation.CancelOtherWithLabel(ctx, motionOpId)
+
+	// create a new empty framesystem which we will add our base to
+	fs := referenceframe.NewEmptyFrameSystem("")
+
+	// build maps of input enabled resources
+	_, resources, err := ms.fsService.CurrentInputs(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	// get the localizer
+	local, ok := ms.localizers[movementSensorName]
+	if !ok {
+		return false, resource.DependencyNotFoundError(movementSensorName)
+	}
+
+	// assert localizer as a movementSensor to get map limits
+	movementSensor, ok := local.(movementsensor.MovementSensor)
+	if !ok {
+		return false, fmt.Errorf("cannot assert local of type %T as a movementSensor", local)
+	}
+
+	currentPosition, _, err := movementSensor.Position(ctx, nil)
```

Comment:
you don't need to cast this to a movement sensor, the localizer has a position function already

Reply:
true that!